### PR TITLE
Resolves #1598 : Added responsive dropdown at sessions, tickets and events dashboard.

### LINKED ISSUE #1598: Responsive model for Tabs
We need to decide on a responsive tab design. Does anyone have any suggestion for tabs in small devices.. The model right now doesn't work very good.

### PR DESCRIPTION
**Pull request by sainiak009**
I have replaced all responsive tabs with dropdowns. Please review and merge. @SaptakS @niranjan94 
screenshot 30
screenshot 31
screenshot 33

### CODE REVIEW

File: app/templates/gentelella/admin/mysessions/mysessions_list.html [hunk 1]
```diff
@@ -1,105 +1,115 @@
...
-                {% set vars = {'foo': False} %}
-                {% for custom_image in custom_placeholder %}
-                    {% if custom_image.name == session.event.topic %}
-                        <img src='{{ custom_image.url }}' style="height: 88%">
-                        {% if vars.update({'foo': True}) %} {% endif %}
-                    {% endif %}
-                {% endfor %}
-                {% if not vars.foo %}
-                    <img src='{{ url_for("static", filename="placeholders/thumbnails/" + placeholder_images[session.event.topic]) }}' style="height: 88%">
+            {% set vars = {'foo': False} %}
```
Comment (niranjan94) [inline app/templates/gentelella/admin/mysessions/mysessions_list.html]: Could you give better names to variables and dict keys ?

Comment (niranjan94) [inline app/templates/gentelella/admin/mysessions/mysessions_list.html]: @sainiak009 thanks :)

**Comment by sainiak009** [comment]
@SaptakS @niranjan94  All the required changes are done. Please review and merge.

**Comment by SaptakS** [comment]
@sainiak009 please squash your commits.

**Comment by sainiak009** [comment]
@SaptakS Commits squashed. Please merge.

**Pull request merged**